modules/3d-tiles/src/lib/parsers/helpers/normalize-3d-tile-normals.ts

```diff
@@ -1,16 +1,20 @@
 import {Vector3} from '@math.gl/core';
 import {GL, octDecode} from '@loaders.gl/math';
+import {Tiles3DTileContent} from 'modules/3d-tiles/src/types';
```

Comment:
Import from relative files please. (I believe you can turn off the vscode feature that automatically adds these incorrect imports)